pylint/checkers/classes/class_checker.py

```diff
@@ -797,6 +797,7 @@ def _ignore_mixin(self):
         "duplicate-bases",
         "redefined-slots-in-subclass",
         "invalid-enum-extension",
+        "subclassed-final-class",
```

Comment:
I think we could add this change on main first so the CI is faster when we modify the documentation.

Reply:
Completed in https://github.com/PyCQA/pylint/pull/6190.